### DISCUSSION TITLE
End of life support for rxjs v7.5.0

### DISCUSSION THREAD

**Discussion by pneumowave-finlay-mcsporran** [ReactiveX/rxjs#7500] [Q&A]
Hi,

We are in the process of applying for FDA certification and need to include information about the end-of-life dates for packages. Could someone please inform me if there are any available details regarding the end of support for rxjs version 7.5.0?

**Comment by ujamX** [comment]
Hi, @pneumowave-finlay-mcsporran 

There appears to be no official information from the RxJS team at this time regarding the end of support date for RxJS 7.5.0.
It is expected that you will receive a clearer and faster response if you contact the RxJS maintenance team directly.
Best of luck 🍀
